engine/src/main/java/io/camunda/zeebe/engine/state/message/DbProcessMessageSubscriptionState.java

```diff
@@ -70,11 +70,17 @@ public void put(final long key, final ProcessMessageSubscriptionRecord record) {
     processMessageSubscription.reset();
     processMessageSubscription.setKey(key).setRecord(record);
 
-    subscriptionColumnFamily.put(elementKeyAndMessageName, processMessageSubscription);
+    subscriptionColumnFamily.insert(elementKeyAndMessageName, processMessageSubscription);
 
     transientState.add(record);
   }
 
+  @Override
```

Comment:
I added this method to avoid `put`ing the `ProcessMessageSubscriptionRecord` again, here: https://github.com/camunda-cloud/zeebe/blob/main/engine/src/main/java/io/camunda/zeebe/engine/state/migration/DbMigrationState.java#L148-L152

This new method is in line with the other methods here, such as `updateToOpenedState`.
I guess this another change that should have been a seperate commit.